pkg/appconsts/v3/app_consts.go

```diff
@@ -1,10 +1,14 @@
 package v3
 
+import "time"
+
 const (
 	Version              uint64 = 3
 	SquareSizeUpperBound int    = 128
 	SubtreeRootThreshold int    = 64
 	TxSizeCostPerByte    uint64 = 10
 	GasPerBlobByte       uint32 = 8
 	MaxTxSize            int    = 2097152 // 2 MiB in bytes
+	TimeoutPropose              = time.Second * 3
+	TimeoutCommit               = time.Second * 1
```

Comment:
[To the reviewers] We may need further research to properly set these values.

Reply:
By default, we should use the timeouts from the 8MB 6s chain, so 4.2s timeout commit and 3.5s timeout propose. This gets us 5-7s blocks for a 0-8MB block

If we want closer to 6s second blocks we can adjust for testnets and max block size